tests/main/snap-run/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that `snap run` runs
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local basic-run
+
+restore: |
```

Comment:
I believe that we don't need to remove those snaps as `install_local` handles that now.